### PR TITLE
Prepare version 1.0.0

### PR DESCRIPTION
**Pull request by tobiasge**
Related Issue:- 

## New Behavior
- Serving Netbox with Ngin Unit
## Contrast to Current Behavior
- Gunicorn and extra nginx were used
## Changes to the Wiki
- See PR #361

## Proposed Release Note Entry

```md
# Noteworthy Changes

This release re-works how we serve HTTP content and introduced more initializers.

## Migrate from _Nginx & Gunicorn_ to _Nginx Unit_ #361

We've changed how we serve Netbox's HTTP traffic fundamentally. Instead of an `nginx` and a `netbox` container, it's now just a `netbox` container. And instead of relying on `nginx` to server static content and `gunicorn` to serve dynamic content, the Nginx `unit` application server does it all.

This makes the whole setup less brittle and adds a few benefits like faster startup time and simpler deployments.

Big thanks to @tobiasge for the work on this!

### 🚨 Upgrading from Earlier Versions? Read this! ⚠️

The `nginx` container was removed entirely. If you reference it in your `docker-compose.override.yml`, make sure to change `nginx` to `netbox`:

\```patch
  version: '3.4'
  services:
-   nginx:
+   netbox:
      ports:
      - '8080:8080'
\```

Also, if you build a container based on Netbox Docker, please be aware that we now rely on a virtual env to install our Python packages. 

## More Initializers #384 

@ryanmerolle contributed a whole bunch of new initializers:

- Circuit Types
- Circuits
- Cluster Groups
- Clusters
- Power Feeds
- Power Panels
- Providers
- Route Targets
- Secret Roles
- Services
- Tags

Thank you @ryanmerolle a lot!

## Fix Assignment of Primary IPs in Initializers #393

@MajesticFalcon discovered that it is sometimes not possible to assign a primary ip and provided a fix for this.

There was an issue that `LOGIN_TIMEOUT` could not be defined.

## Introducing our Github Community #370

Github recently made the Discussions feature available in Beta. [We've opted to try this feature](https://github.com/netbox-community/netbox-docker/discussions) in order to lower the barrier for getting help. For now, the NetworkToCode Slack will remain available and active.

# Compatibility

This version of Netbox Docker is only compatible with Netbox v2.10.x. For older versions use [a previous release of Netbox Docker](https://github.com/netbox-community/netbox-docker/releases/).

# Known Issues

There are no confirmed issues at the time of writing this. [See the issues labelled _bug_ in our issues section](https://github.com/netbox-community/netbox-docker/issues?q=is%3Aissue+is%3Aopen+label%3Abug).

# Sponsoring

Consider buying the maintainers of this project a beer or drink:

- [Sponsor @cimnine](https://github.com/sponsors/cimnine)
- [Sponser @tobiasge](https://github.com/sponsors/tobiasge)
```

## Double Check
* [x] I have read the comments and followed the PR template.
* [x] I have explained my PR according to the information in the comments.
* [ ] My PR targets the `develop` branch.

### CODE REVIEW

**Comment by cimnine** [comment]
I would like to also have https://github.com/netbox-community/netbox-docker/pull/391 in here as well, so I have not approved yet. But this will become another kick-ass release 💪 :)

**Comment by cimnine** [comment]
I was just thinking whether this should become a `1.0.0` instead of `0.28.0`? What do you think?

**Comment by cimnine** [comment]
I've took the liberty to update your description with a draft for the release notes.

**Comment by tobiasge** [comment]
> I was just thinking whether this should become a `1.0.0` instead of `0.28.0`? What do you think?

This is a big change and the new image won't be compatible with old compose and configuration files. I like the idea of a 1.0 release.

**Review by cimnine — APPROVED** [review]
Are we ready for this? 😇

**Comment by tobiasge** [comment]
> Are we ready for this? 😇

I think so.

**Comment by danpoltawski** [comment]
Just to note that it's not clear from the documentation that you can't use this docker-compose configuration with older versions of netbox (I was trying to create a local environment matching our production env (v2.9.x) and this change caught me out)

**Comment by cimnine** [comment]
> Just to note that it's not clear from the documentation that you can't use this docker-compose configuration with older versions of netbox

I disagree. [Our release notes](https://github.com/netbox-community/netbox-docker/releases) clearly state which release of Netbox Docker works with which release of Netbox, e.g. today's version 1.0.2:

> # Compatibility
> 
> This version of Netbox Docker is only compatible with Netbox v2.10.x. For older versions, use a previous release of Netbox Docker.

Also, since a few hours ago, this exists with an improved wording: https://github.com/netbox-community/netbox-docker/wiki/Version

**Comment by danpoltawski** [comment]
Ok, let me rephrase. It's possible (and I decided to comment here because I spent some time trying to work it out myself) to get to the front page of the project/ read the docker hub page and see the note about  `vX.Y.Z: These are release builds, automatically built from the corresponding releases of NetBox.` and not realise that older versions need a quite different configuration. As it's my first experience of using the project I did not think to check the release notes, as it was my first experience (I did note the version numbs but because they aren't aligned to netbox versions I didn't realise what was compatible with what).

Thanks for your work on this project :-)
